Tasks/ConsoleApp/AsyncAwait/ProductsContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+namespace AsyncAwait
+{
+    public class ProductsContext : DbContext
+    {
+        public DbSet<Product> _products { get; set; }
```

Comment:
Wrong code convention. If it is a public Property then name should be in Pascal case **Products**, if it's private property then **_products**